packages/sources/intrinio-test/src/config/index.ts

```diff
@@ -0,0 +1,17 @@
+export const customSettings = {
+  API_ENDPOINT: {
+    description: 'The API url for intrinio',
+    type: 'string',
+    default: 'https://api-v2.intrinio.com/',
+  },
+  API_KEY: {
+    description: 'The API key for intrinio',
+    type: 'string',
+    require: true,
+  },
```

Comment:
Should we make this sensitive?

Reply:
done :)

Reply:
Hmm, did you push up your changes? I don't see that here